### PR TITLE
Styleguide fixes for Windurst Woods and usage of npcUtils

### PR DESCRIPTION
**Pull request by kaincenteno**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

would love a second set of eyes to take a look at it in case i messed something up, but this is only styleguide fixes and replacement of additem to use npcutil.giveitem

### CODE REVIEW

**Comment by kaincenteno** [comment]
changes discussed on discord earlier were done. Ready for review, but would need someone to test >.<

File: scripts/zones/Windurst_Woods/npcs/Anillah.lua [hunk 1]
```diff
@@ -32,7 +32,10 @@ function onEventUpdate(player, csid, option)
 end
 
 function onEventFinish(player, csid, option)
-    if csid == 10015 and option == 1 then
+    if
+        csid == 10015 and
```
Comment (TeoTwawki) [inline scripts/zones/Windurst_Woods/npcs/Anillah.lua:36]: > [Try not to exceed 120 chars width. Exceptions will occur, but try.](https://github.com/project-topaz/topaz/blob/release/CONTRIBUTING.md#general-code-guidlines-all-languages)

.

> [-- Long or many multiple conditions - Correct ✔️](https://github.com/project-topaz/topaz/blob/release/CONTRIBUTING.md#formatting-conditional-blocks)


Notice how this was not long nor was it many. Already talked to you about this before. so ... REEEEEEEEEEEEEEEEeee :stuck_out_tongue:

File: scripts/zones/Windurst_Woods/npcs/Nhobi_Zalkia.lua [hunk 2]
```diff
@@ -16,17 +16,17 @@ end
...
+        4412,  299,  -- Thundermelon
+        4491,  184   -- Watermelon
+    }
+
     if RegionOwner ~= tpz.nation.WINDURST then
         player:showText(npc, ID.text.NHOBI_ZALKIA_CLOSED_DIALOG)
     else
         player:showText(npc, ID.text.NHOBI_ZALKIA_OPEN_DIALOG)
-
-        local stock =
```
Comment (TeoTwawki) [inline scripts/zones/Windurst_Woods/npcs/Nhobi_Zalkia.lua:24]: you know this isn't used outside this block, theres no need to move it.

**Comment by kaincenteno** [comment]
REEEEEEEEEeee @TeoTwawki

**Pull request closed without merging**